sample-dcrctl.conf

```diff
@@ -34,3 +34,20 @@
 ; RPC server certificate chain file for validation
 ; rpccert=~/.dcrd/rpc.cert
 
+; Disable TLS
+; notls=0
```

Comment:
This (and the other cases) should really be set to 1 so that uncommenting the directive does what the comment says it will.  That is, `notls=1` will disable TLS.